distributed/http/semaphore/prometheus.py

```diff
@@ -0,0 +1,112 @@
+import numpy as np
+
+
+class SemaphoreMetricExtension:
+    def __init__(self, dask_server):
+        self.server = dask_server
+
+    def collect(self):
+        from prometheus_client.core import (
```

Comment:
why the local import?

Reply:
Same as in `http/scheduler/prometheus.py` (and `worker`). I assumed there must be some reason for having it there. Perhaps somebody else can shine some light on that

Reply:
probably about having prometheus as an optional dependency. That's fine I guess.